commons-app/pom.xml

```diff
@@ -34,6 +30,15 @@
 			<name>SpringSource Enterprise Bundle Repository - SpringSource Bundle Releases</name>
 			<url>http://repository.springsource.com/maven/bundles/release</url>
 		</repository>
+
+		<repository>
+			<id>3rdParty</id>
```

Comment:
@cmader What is the purpose of this repository - which artifacts do we need to get from there?